src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/repository/CourtOrderRepository.kt

```diff
@@ -8,7 +8,7 @@ import uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.OffenderBooking
 
 @Repository
 interface CourtOrderRepository : JpaRepository<CourtOrder, Long> {
-  fun findByOffenderBookingAndCourtEventAndOrderType(
+  fun findFirstByOffenderBookingAndCourtEventAndOrderTypeOrderByIdAsc(
     offenderBooking: OffenderBooking,
```

Comment:
there are 115 duplicates in live; just select the first one - it actually doesn't matter which one is selected so long as there is one